omniNotes/src/androidTest/java/it/feio/android/omninotes/helpers/HelpersMethods.java

```diff
@@ -0,0 +1,147 @@
+        int leftLimit = 97; // letter 'a'
+        int rightLimit = 122; // letter 'z'
+        int targetStringLength = 10;
+        Random random = new Random();
+        StringBuilder buffer = new StringBuilder(targetStringLength);
+        for (int i = 0; i < targetStringLength; i++) {
+            int randomLimitedInt = leftLimit + (int)
+                    (random.nextFloat() * (rightLimit - leftLimit + 1));
+            buffer.append((char) randomLimitedInt);
+        }
+        return buffer.toString();
+    }
+
+    public static void addItemsContent(Matcher<View> matcher, int contentTextLines) {
+        String noteContent = generatingRandomText();
+        for (int i = 0; i < contentTextLines; i++) {
+            onView(matcher).perform(typeText(noteContent));
+            onView(matcher).perform(pressKey(66));
+        }
+    }
+
+    public static boolean isUIObjectIsClickable(Matcher<View> matcher) {
+        try {
+            onView(matcher).check(matches(isClickable()));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectAboveAnotherObject(Matcher<View> matcher1, Matcher<View> matcher2) {
+        try {
+            onView(matcher1).check(isAbove(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectBelowAnotherObject(Matcher<View> matcher1, Matcher<View> matcher2) {
+        try {
+            onView(matcher1).check(isBelow(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectRightOfAnotherUIObject(Matcher<View> matcher1, Matcher<View> matcher2) {
+        try {
+            onView(matcher1).check(isRightOf(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectLeftOfAnotherUIObject(Matcher<View> matcher1, Matcher<View> matcher2) {
+        try {
+            onView(matcher1).check(isLeftOf(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static void addNoteItems(Matcher<View> matcher, String specificToDoItemText, int position, int noOfItems) {
+        for (int i = 0; i < noOfItems; i++) {
+            Home.clickAddButton();
+            Home.clickTextNoteButton();
+            if (i == position) {
```

Comment:
You could name "position" more specific like positionOfSpecificItem.

Reply:
Done.